manifests/nodejs.yml

```
@@ -355,6 +355,7 @@ tests/:
       Test_Main: v2.0.0
     test_remote_config_rule_changes.py:
       Test_BlockingActionChangesWithRemoteConfig: *ref_4_1_0
+      Test_UpdateRuleFileWithRemoteConfig: missing_feature
```

same version as when ASM_DD was implemented
```suggestion
      Test_UpdateRuleFileWithRemoteConfig: *ref_3_19_0
```